### PR TITLE
added import method and cluster_yaml argument

### PR DESCRIPTION
**Pull request by rawmind0**
This PR contains:
- Deprecated fields `internal_kube_config_yaml` and `node_conf`
- Added `cluster_yaml` argument
- Added import method to `rke_cluster` resource
- Fixed `vsphere_cloud_provider` argument to avoid false diff
- Fixed issues #184, #181  and #75 
- rke updated to v1.0.6
- Updated documentation and CHANGELOG.md

### CODE REVIEW

File: website/docs/r/cluster.html.markdown [hunk 1]
```diff
@@ -10,18 +10,26 @@ description: |-
...
 Creating RKE cluster
 
 ```hcl
 # Configure RKE provider
 provider "rke" {
-  debug = true
   log_file = "rke_debug.log"
 }
-# Create a new RKE cluster
+Create a new RKE cluster using config yaml
```
Comment (cloudnautique) [inline website/docs/r/cluster.html.markdown:27]: should add the # for a comment.

File: website/docs/r/cluster.html.markdown [hunk 2]
```diff
@@ -10,18 +10,26 @@ description: |-
 
 Provides RKE cluster resource. This can be used to create RKE clusters and retrieve their information.
 
+RKE clusters can be configured in distinct ways:
+- Using `cluster_yaml` argument: On this mode, can use a full RKE cluster config yaml file.
+- Using provider arguments: On this mode, RKE config is generated with arguments like `nodes`, `services`,...,etc
+- Using both: On this mode, can use a full RKE cluster config yaml file and provider arguments. Arguments has preference.
```
Comment (cloudnautique) [inline website/docs/r/cluster.html.markdown:16]: We should have an example of both.

File: website/docs/r/cluster.html.markdown [hunk 3]
```diff
@@ -10,18 +10,26 @@ description: |-
 
 Provides RKE cluster resource. This can be used to create RKE clusters and retrieve their information.
 
+RKE clusters can be configured in distinct ways:
+- Using `cluster_yaml` argument: On this mode, can use a full RKE cluster config yaml file.
```
Comment (cloudnautique) [inline website/docs/r/cluster.html.markdown:14]: Using `cluster_yaml`: The full RKE cluster is defined in an RKE cluster.yml file.
Using the TF provider arguments to define the entire cluster.
Using a combination of both the `cluster_yaml` and TF provider arguments. The TF arguments will override the `cluster_yaml` options if collisions occur.

File: website/docs/r/cluster.html.markdown [hunk 4]
```diff
@@ -10,18 +10,26 @@ description: |-
 
 Provides RKE cluster resource. This can be used to create RKE clusters and retrieve their information.
 
+RKE clusters can be configured in distinct ways:
```
Comment (cloudnautique) [inline website/docs/r/cluster.html.markdown:13]: RKE clusters can be defined in the provider:

File: website/docs/r/cluster.html.markdown (hunk 1 above)
Comment (rawmind0) [reply website/docs/r/cluster.html.markdown:27]: Fixed

File: website/docs/r/cluster.html.markdown (hunk 2 above)
Comment (rawmind0) [reply website/docs/r/cluster.html.markdown:16]: Added both example

File: website/docs/r/cluster.html.markdown (hunk 3 above)
Comment (rawmind0) [reply website/docs/r/cluster.html.markdown:14]: Updated

File: website/docs/r/cluster.html.markdown (hunk 4 above)
Comment (rawmind0) [reply website/docs/r/cluster.html.markdown:13]: Updated

**Pull request merged**